ubuntu-1804-image_5b461f730428630abc0bf442.md

```diff
@@ -69,32 +69,35 @@ Docker toolset is installed and following versions are available:
 ## Databases and services
 
 Databases run in the same environment as your code. You can start a local
-database service using the [sem-service][sem-service] tool:
+database service using the [sem-service](https://docs.semaphoreci.com/article/54-toolbox-reference#sem-service)
+tool:
 
 ```
 sem-service start mysql
 sem-service start postgres
 sem-service start redis
 sem-service start memcached
+sem-service start mongodb
+sem-service start elasticsearch
 ```
 
 Credentials:
 
-- MySQL username is `root` and password is blank.
-- PostgreSQL username is `postgres` and password is blank.
+- The default MySQL username is `root` and password is blank.
+- The default PostgreSQL username is `postgres` and password is blank.
 
 Services are not automatically shared across blocks. To do that, start services
-within the [prologue][prologue] of each task and populate data as needed.
+within the [prologue](https://docs.semaphoreci.com/article/50-pipeline-yaml#prologue)
+property of each task and populate data as needed.
 
 ## Languages
 
 ### Erlang and Elixir
 
-Erlang versions are installed and managed via
-[kerl](https://github.com/kerl/kerl). Elixir versions are installed with
-[kiex](https://github.com/taylor/kiex).
+Erlang versions are installed and managed via [kerl](https://github.com/kerl/kerl).
+Elixir versions are installed with [kiex](https://github.com/taylor/kiex).
 
-- Erlang: 21.0
+- Erlang: 20.0, 21.0
```

Comment:
the erlag 20.0 is actually 20.3

Reply:
OK - I am changing that to 20.3. Thanks!